handsontable/src/core/hooks/constants.js

```diff
@@ -620,6 +631,17 @@ export const REGISTERED_HOOKS = [
    */
   'afterRenderer',
 
+  /**
+   * Fired after the cache of the row sequence has been updated.
+   *
+   * @event Hooks#afterRowSequenceCacheUpdate
```

Comment:
```suggestion
   * Fired after the cache of the row sequence has been updated.
   *
   * @since 16.2.0 
   * @event Hooks#afterRowSequenceCacheUpdate
```